misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormatterUtils.java

```diff
@@ -526,8 +561,12 @@ private static Indentation getBlockParent(Node node) {
         if (parent.kind() == SyntaxKind.IF_ELSE_STATEMENT) {
             ArrayList nestedBlock = nestedIfBlock((NonTerminalNode) parent);
             if (!nestedBlock.isEmpty()) {
+                boolean addSpaces = false;
+                if (parent.parent() != null && parent.parent().kind() == SyntaxKind.BLOCK_STATEMENT) {
```

Comment:
Shall we assign the addSpaces value with the if's condition? No need to add an if clause just for setting the variable

Reply:
Will make the following improvement in a refactoring PR.
`boolean addSpaces = parent.parent() != null && parent.parent().kind() == SyntaxKind.BLOCK_STATEMENT;`

Reply:
Great, approving the PR upon the above comment